### ISSUE TITLE
Test-only issue with Bokeh 3.0dev

### ISSUE THREAD

**Issue by bryevdv** [dask/dask#9257]
cc @jrbourbeau 

There is a failing Dask test in the Bokeh 3.0 dev "downstream" job:
```
_____________________________________________________________ test_plot_multiple _____________________________________________________________

    @pytest.mark.skipif("not bokeh")
    @pytest.mark.skipif("not psutil")
    def test_plot_multiple():
        from dask.diagnostics.profile_visualize import visualize

        with ResourceProfiler(dt=0.01) as rprof:
            with prof:
                get(dsk2, "c")
        p = visualize(
            [prof, rprof], label_size=50, title="Not the default", show=False, save=False
        )
>       figures = [r[0] for r in p.children[1].children]
E       AttributeError: 'tuple' object has no attribute 'children'

dask/diagnostics/tests/test_profiler.py:333: AttributeError
```

This is because in Bokeh 3.0 a grid plot is a real proper layout object unto itself, and no longer merely a helper function to assemble a ragged "column of distinct rows" array. So `.children[1].children` is no longer a thing. 

I _think_ this is only a test issue, and not a real issue:

* The dashboard seems to  function exactly as expected with some simple smoke tests
* I was not able to find any usage of `.children` in actual library code

**Comment by pavithraes** [comment]
@bryevdv Thanks for opening this! I agree, it does look like just a test issue.

Relevant code: https://github.com/dask/dask/blob/d6c0878472320661f1e1d3e636b422de5565d622/dask/diagnostics/tests/test_profiler.py#L333

Possible fix, we can update it to: `figures = [r[0] for r in p.children]` -- I tested this with `bokeh=3.0.0dev10` and dask=`main` and it seems to work. :)

**Comment by jrbourbeau** [comment]
Thanks for surfacing @bryevdv. I agree with you and @pavithraes that this is a test-only issue. 

> Possible fix, we can update it to: `figures = [r[0] for r in p.children]`

Thanks for the suggestion -- giving it a try over in https://github.com/dask/dask/pull/9261

**Comment by pavithraes** [comment]
#9261 is merged, and [this specific test in Bokeh 3.0's downstream job](https://github.com/bokeh/bokeh/runs/7321433192?check_suite_focus=true#step:5:39) seems to be passing now! 

Thank you, `@jrbourbeau` and `@bryevdv`!

I'll go ahead and close this as resolved, please feel free to re-open it if needed!
